style.css

```diff
@@ -0,0 +1,122 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  line-height: 1;
+  letter-spacing: 1.2;
+}
+
+html {
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  width: 100vw;
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.header-section {
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+  max-width: 100vw;
+  padding-bottom: 40px;
+  background-color: #fff;
+  border-bottom-left-radius: 8rem;
+  background-image: url(./images/header-img.png);
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position: top 8rem right 0;
+}
+
+nav {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 2rem;
+}
+
+.logo {
+  width: 7.6rem;
+  height: 2rem;
+  color: #6070ff;
+  font-size: 1.8rem;
+  font-weight: 700;
+  line-height: 2rem;
+}
+
+ul {
+  display: flex;
+  gap: 3rem;
+  list-style: none;
+}
+
+li {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+}
+
+.bio {
+  margin: 20rem auto;
+}
+
+.heading {
+  max-width: 100vw;
+  max-height: 70vh;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 4rem;
+  line-height: 6rem;
+  color: #344563;
+  margin: 0 1rem;
+}
+
+.paragraph {
+  max-width: 100vw;
+  max-height: 60vh;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.9rem;
+  line-height: 2.8rem;
+  color: #344563;
+  margin: 0 1rem;
+}
+
+.heading-3 {
+  max-width: 100vw;
+  max-height: 100vh;
+  text-transform: uppercase;
+  color: #7f8cff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+  margin: 0 1rem;
```

Comment:
- [ ] As requested above ☝🏽 please use the px unit as in the original version it makes it easy to compare them.